sec-keys/openpgp-keys-apache-zeppelin/metadata.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
-	<maintainer type="person">
-		<email>gentoo@pogatzki.net</email>
-		<name>Volkmar W. Pogatzki</name>
-	</maintainer>
```

Comment:
Please drop `proxy-maint@gentoo.org` too

Reply:
done.

BTW: for maven-bin, i am proposing [this way](https://github.com/gentoo/gentoo/pull/43580)